featuresVar-Italic.fea

```diff
@@ -0,0 +1,10 @@
+include (../../../../familyTables.fea);
```

Comment:
Could this whole file be omitted by using a global featuresVar.fea file, and keeping the STAT feature name generic in Roman/Italic folders?

Reply:
(same thing happens with familyGSUB for example)